src/smtencoding/FStar.SMTEncoding.Encode.fs

```diff
@@ -343,6 +343,42 @@ let is_eta env vars t =
         | _ -> None in
   aux t (List.rev vars)
 
+(* [is_reifiable_* env x] returns true if the effect name/computational effect (of *)
+(* a body or codomain of an arrow) [x] is reifiable *)
+
+let is_reifiable_effect (env:Env.env) (effect_lid:lident) : bool =
+  let quals = Env.lookup_effect_quals env effect_lid in
+  List.contains Reifiable quals
+
```

Comment:
This utility seems generally useful. How about moving it into FStar.TypeChecker.Env?